pipeline/custom.groovy

```diff
@@ -0,0 +1,162 @@
+import groovy.io.FileType
+def nodeName = "centos-7"
+def sharedLib
+
+def GetCLI(args){
+	/*
+		Generates the CLI using the arguments provided and returns it.
+	*/
+	echo "Inside get cli"
+	def cli = 'cd $WORKSPACE; .venv/bin/python run.py'
+	cli += ' --rhbuild ' + args['rhbuild']
+	cli += ' --platform ' + args['platform']
+	cli += ' --build ' + args['build']
+	cli += ' --v2'
+	cli += ' --global-conf ' + args['global_conf']
+	cli += ' --osp-cred ~/osp-cred-ci-2.yaml'
+	cli += ' --inventory ' + args['inventory']
+	cli += ' --suite ' + args['suite_name']
+	cli += ' --instances-name ' + args['instances_name']
+	cli += ' ' + args['addn_args']
+
+	if(cli.indexOf('log-level') < 0){
+	    cli += ' --log-level info'
+	}
+
+	if(args['rhs_ceph_repo']?.trim() || args['container_image']?.trim()){
+		cli += ' --ignore-latest-container'
+	}
+	if(args['rhs_ceph_repo']?.trim()){
+		cli += ' --rhs-ceph-repo ' + args['rhs_ceph_repo']
+	}
+	if(args['add_repo']?.trim()){
+		cli += ' --add-repo ' + args['add_repo']
+	}
+	if(args['hotfix_repo']?.trim()){
+		cli += ' --hotfix-repo ' + args['hotfix_repo']
+	}
+	if(args['container_image']?.trim()){
+		container = args['container_image'].split('.com/')
+		docker_registry = container[0] + '.com'
+		docker = container[1].split(':')
+		docker_tag = docker.last()
+		docker_image = docker.first()
+		cli += ' --insecure-registry'
+		cli += ' --docker-registry ' + docker_registry
+		cli += ' --docker-tag ' + docker_tag
+		cli += ' --docker-image ' + docker_image
+	}
+	echo "cli"
+	echo cli
+	return cli
+}
+
+def ExecuteBuildStage(instances_name){
+    echo "Inside execute build stages"
+    Ceph_Version = 'pacific'
```

Comment:
could we use groovy style variable name i.e. `cephVersion`?

Reply:
Done.